### PR TITLE
Cleanup setup and requirements

### PR DESCRIPTION
**Pull request by itziakos**
This PR tries to fix the requirements for kiva and enable and at the same time clean-up the travis-ci build

### CODE REVIEW

**Comment by itziakos** [comment]
Please note that now the PR tests take only 7 minutes instead of 12-14.

File: enable/__init__.py [hunk 1]
```diff
@@ -3,10 +3,11 @@
 """ A multi-platform object drawing library.
     Part of the Enable project of the Enthought Tool Suite.
 """
-
-from kiva._version import full_version as __version__
+import sys
```
Comment (jwiggins) [inline enable/__init__.py]: Not used here

**Comment by jwiggins** [comment]
Other than that unused import I'm :+1: on this PR

**Comment by jwiggins** [comment]
Go ahead and merge as soon as Travis gives the OK. I'm going offline for a bit.

**Comment by itziakos** [comment]
Ok we now pass the setup.py import and we arrive at a new issue which now is the result of the interaction between setuptools and numpy.distutils. It looks that if any packages that is in the `install_requires` list it will result in an out  of memory error when that required packages are installed.

So, that is why commit 14e11492 fails, because it will cause the required packages to be downloaded by the combined setuptools/numpy.distutils setup.py. I have been able to replicate the behaviour locally.

**Comment by itziakos** [comment]
So it looks that the last version of setuptools that work is 9.1

**Pull request merged**